exercises/change/cases_test.go

```diff
@@ -0,0 +1,126 @@
+package change
+
+// Source: exercism/x-common
+// Commit: 3d8b5b3 change: Fix canonical-data.json formatting
+
+var testCases = []struct {
+	description string
+	coins       []int
+	target      int
+	expected    interface{}
```

Comment:
I think it would be good if we could have an `[]int` and a `bool` here, rather than an `interface{}`, since otherwise it may not be clear to the student what type the function should return